lib/commons/dom/find-up.js

```diff
@@ -9,33 +9,24 @@
  * @return {HTMLElement|null} Either the matching HTMLElement or `null` if there was no match
  */
 dom.findUp = function (element, target) {
-	'use strict';
```

Comment:
These changes don't seem to actually change anything in terms of functionality - is this purely a performance fix?